### PR TITLE
Document how to create API ML extensions and SAF IDT implementation

### PR DESCRIPTION
**Pull request by taban03**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: versioned_docs/version-v1.19.x/extend/extend-apiml/implement-new-saf-provider.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+     * Verify that the provided saf token is valid.
+     *
+     * @param safToken Token to validate.
+     * @return true if the token is valid, false if it is invalid
+     */
+    boolean verify(String safToken);
+}
+```
+
+The `SafIdtProvider` interface contains the `generate` and `verify` methods, that can be overridden by your SAF IDT implementation
```
Comment (janan07) [inline versioned_docs/version-v1.19.x/extend/extend-apiml/implement-new-saf-provider.md]: Check this for accuracy: The `SafIdtProvider` interface contains the `generate` and `verify` methods. These methods can be overridden by your SAF IDT implementation to generate the SAF token on behalf of the specified user. This also verifies that the provided SAF token is valid.

Comment (janan07) [reply versioned_docs/version-v1.19.x/extend/extend-apiml/implement-new-saf-provider.md]: Also, what is verifying that the SAF token is valid?

Comment (taban03) [reply versioned_docs/version-v1.19.x/extend/extend-apiml/implement-new-saf-provider.md]: The SAF IDT implementation verify that the SAF token is valid. I think that your reformulation is correct

**Review by janan07 — APPROVED** [review]
Reviewed and edited by Tech Info

**Pull request merged**